### DISCUSSION TITLE
Requesting a matrix Group

### DISCUSSION THREAD

**Discussion by Absinthe8383** [raphamorim/rio#71] [General]
A matrix would be good for communication and people who don't use github can also contribute and that would also be good for support. I didn't request a Discord Server because it is not secure and why not take advantages of matrix if we are just starting out.

**Comment by raphamorim** [comment]
Hey there @Absinthe8383 , thanks for the issue. If more people feel the same, we can validate that option. I will let this issue open for discussion.

**Reply by auronandace** [reply]
Thanks for leaving this open for discussion. Here is my take on the issue:

Discord is both closed source and interested in making money from you using their free service (by serving you ads or mining your personal information). Continuing to use their service is encouraging such behaviour and propping up a closed source project.

Matrix is both open source and mature enough to be a valid alternative. In general I prefer open source projects to use open source communication.

I appreciate that as a developer you would need to choose what is best for you and your community. As someone else has mentioned in another comment, you can bridge Discord and Matrix but obviously someone would need to maintain that.

Thanks for the excellent project and I hope it remains fun for you.

**Comment by avelino** [comment]
> Discord Server because it is not secure 

@Absinthe8383 why isn't discord safe, because it is a private company?

**Reply by Absinthe8383** [reply]
In terms of privacy, the chats are not encrypted, that's why I suggested matrix over discord you are free to choose.

**Reply by OlshaMB** [reply]
Public matrix rooms are also not encrypted, also most users don't know how to use matrix.
I understand that privacy is a big concern especially in discord but this is a public group.
But at the same time how safe it is to store data on discord(especially with recent social media trends(reddit, twitter))

**Reply by OlshaMB** [reply]
We could bridge matrix and discord
